svc/pkg/cluster/worker/src/workers/server_install/install_scripts/files/rivet_fetch_tls.sh

```diff
@@ -0,0 +1,67 @@
+# Write script
+cat << 'EOF' > /usr/bin/rivet_fetch_tls.sh
+#!/usr/bin/env bash
+set -eu
+
+CERT_ID="job"
+STUB="/etc/__NAME__/tls/$CERT_ID"
+
+# Retry script every 5 seconds
+while true; do
+  response=$(
+    curl -f \
+      -H "Authorization: Bearer __SERVER_TOKEN__" \
+      "https://__DOMAIN_MAIN_API__/provision/datacenters/___DATACENTER_ID___/tls"
```

Comment:
Need to be done via tunnel the same way as api-route. Should not be exposed publicly. Should probably be api-tls or something. You can create a private monolith for both api-route and api-tls.

Reply:
Already done in a future PR